README.md

```diff
@@ -1,8 +1,35 @@
 # local-storage-operator
 Operator for local storage
 
+```mermaid
+graph LR
+    PVC -->|Requests local storage using a StorageClass| StorageClass
+    StorageClass -->|Defines a LocalVolume| LocalVolume
+    LocalVolume -->|Represents a physical storage device already mounted on a node| Node
```

Comment:
LocalVolume does not need to be mounted on a node, it can be just a block device.

Reply:
Do you have a recommendation about what we should state as the relationship between the localvolume and the node? Right now I just have "Represents a physical storage device already mounted/existent on a node".